R/with_parameters.R

```diff
@@ -156,11 +156,22 @@ build_label <- function(..., case_names) {
 }
 
 build_and_run_test <- function(..., .test_name, desc, code, env) {
-  completed_desc <- paste(desc, .test_name)
   args <- list(..., .test_name = .test_name)
+  completed_desc <- glue_data(args, desc)
```

Comment:
Re: https://github.com/r-lib/lintr/issues/2706, I think your suggested changes make sense. That said, it does seem like we've ~inadvertently set up a rake to step on -- anyone writing `{...}` in a test name for whatever reason (e.g. a package name in curly-style?) can now get mysterious test failures from {patrick} itself.

Maybe it makes sense to wrap this in a `tryCatch()` and throw a warning? ("glue interpretation of test with description "..." failed; interpreting it literally. Use escapes to avoid this warning") And/or possibly add a parameter to explicitly disable the glue interpretation? WDYT?